### DISCUSSION TITLE
Change WASM scale/resolution

### DISCUSSION THREAD

**Discussion by HavenDV** [unoplatform/uno#8142] [Q&A]
When I turn on debug for WebAssembly, the application starts to run in a different resolution/scale. And I like it better because the standard scale is quite large. Is there any way to manage this?
My display settings:
<img width="768" alt="image" src="https://user-images.githubusercontent.com/3002068/154831423-f6af2e0a-6835-485b-a2c3-9aa110c064cb.png">

Repro:
[ratbuddyssey-master (1).zip](https://github.com/unoplatform/uno/files/8122723/ratbuddyssey-master.1.zip)
1. Build Ratbuddyssey.Apps.Uno.WebAssembly.

Debug is disabled:
![image](https://user-images.githubusercontent.com/3002068/155281384-20a5ef12-aa1b-4d14-b630-f3f2f1ad84ce.png)

Debug is enabled(uncomment `"inspectUri": "{wsProtocol}://{url.hostname}:{url.port}/_framework/debug/ws-proxy?browser={browserInspectUri}",`):
![image](https://user-images.githubusercontent.com/3002068/155281710-e9521e55-da9f-4127-b272-a9d7f0c097e0.png)

**Comment by jeromelaban** [comment]
I'm not sure to understand what you mean here. Can you make a video of the behavior?

**Reply by HavenDV** [reply]
I updated the post, added screenshots and repro

**Reply by jeromelaban** [reply]
This is an unknown behavior, not reproduced on our end and possibly related to some visual studio integration. Can you try validating if Blazor shows the same issue ?

**Reply by HavenDV** [reply]
I noticed that when Debug mode is enabled, Chrome is used in guest mode. And when turned off - my standard account with my settings (including the page scaling settings, which is set to 150%). This is what caused the difference
